packages/devtools/lib/src/memory/memory.dart

```diff
@@ -123,11 +119,12 @@ class MemoryScreen extends Screen with SetStateMixin {
             ..layoutHorizontal()
             ..clazz('align-items-center')
             ..add(<CoreElement>[
-              div(c: 'btn-group flex-no-wrap margin-left')
+              div(c: 'btn-group flex-no-wrap')
                 ..add(<CoreElement>[
                   pauseButton,
                   resumeButton,
                 ]),
+              div()..flex(),
```

Comment:
What are UI for button areas defintion to the left and the far right?  I see both ways.  I looked at these buttons as actions more often (snapshot, reset, and pause) so grouping together made more sense?

Reply:
Generally, I think we want to use the left area for frequent or more important sets of actions, and the right area for auxiliary, or 'settings' style actions.

That may change page by page however, so that we can have visually balanced pages, and have similar types of buttons in similar positions between pages.

If you think the buttons on this page make more sense in different grouping or layouts, please do change them back.